apps/experiments/views/experiment.py

```diff
@@ -837,7 +837,10 @@ def get_message_response(request, team_slug: str, experiment_id: int, session_id
     message_details = {"message": None, "error": False, "complete": progress["complete"]}
     if progress["complete"] and progress["success"]:
         result = progress["result"]
-        message_details["message"] = ChatMessage.objects.get(id=result["message_id"])
+        if result["message_id"]:
```

Comment:
The return type for `get_ai_message_id` also includes `None`, so I added this check.